src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -408,6 +409,657 @@ public void RecordParsingAmbiguities()
             EOF();
         }
 
+        [Fact, WorkItem(45538, "https://github.com/dotnet/roslyn/issues/45538")]
+        public void RecordParsing_ConstraintAndSemiColon()
```

Comment:
>RecordParsing_ConstraintAndSemiColon [](start = 20, length = 36)

:memo: by reverting my code change, I confirmed that the only three tests that are affected by the change are record scenarios:
- RecordParsing_ConstraintAndSemiColon
- RecordParsing_ConstraintAndSemiColon_MissingColon
- RecordParsing_TwoConstraintsAndSemiColon